iOS/Settings/SettingsView.swift

```diff
@@ -13,7 +13,7 @@ struct SettingsView: View {
                 Section(header: Text("Appearance")) {
                     PreferencesView(preferences: model.preferences)
                 }
-                Section(header: Text("Support Links")) {
+                Section(header: Text("External Links")) {
```

Comment:
Just curious: what was the rationale for changing the verbiage here?

Reply:
Largely that the collection of links in that section is not _entirely_ support-related (i.e., writing a review for the app). Happy to change it back if you think that's being overly pedantic, though. 🙂

Reply:
Ah, didn't realize this was just the header at first. That makes sense! All good then, thanks.